<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="windowTitle">
   <string>Nuevo Registro</string>
  </property>
  <layout class="QFormLayout">

   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fecha y hora:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>

   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Peso (kg):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="weightSpin"/>
   </item>

   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Presi贸n Sist贸lica:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="systolicSpin"/>
   </item>

   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Presi贸n Diast贸lica:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="diastolicSpin"/>
   </item>

   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Glucosa:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="glucoseSpin"/>
   </item>

   <item row="5" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
